queries.sql

```diff
@@ -1,3 +1,27 @@
-/*Queries that provide answers to the questions from all projects.*/
+SELECT * 
+FROM animals
+WHERE name LIKE '%mon';
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name 
+FROM animals
+WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT date_of_birth
+FROM animals
+WHERE name IN ('Agumon' , 'Pikachu');
+
+SELECT name , escape_attempts
+FROM animals
+WHERE weight_kg > 10.5;
+
+SELECT * 
+FROM animals
+WHERE neutered = true;
+
+SELECT * 
+FROM animals
+WHERE name != 'Gabumon';
+
+SELECT *
+FROM animals
+WHERE weight_kg BETWEEN 10.4 AND 17.3;
```

Comment:
- [ ] Query for **_it can list the name of all animals that are neutered and have less than 3 escape attempts_** is missing. Kindly add the same, as well as the screenshot of the query result in the Pull Request description

Reply:
Alright!